frontend/test/__support__/e2e/helpers/e2e-boolean-helpers.js

```diff
@@ -0,0 +1,24 @@
+// until we have a test dataset that includes boolean data, we can use this questions to test booleans
+const BOOLEAN_QUERY =
+  'select 0::integer as "integer", true::boolean AS "boolean" union all \nselect 1::integer as "integer", false::boolean AS "boolean" union all \nselect null as "integer", true::boolean AS "boolean" union all \nselect -1::integer as "integer", null AS "boolean"';
+
+export const setupBooleanQuery = () => {
```

Comment:
Nit: While I don't have a strong preference for the code style, most of other helpers were written with the `function` syntax rather than the `const`.

If we import them from "global" helpers it doesn't really matter. The only reason I was using `function`s when (initially) writing helpers inline with the test was because of hoisting. It felt nice to remove the "noise" to the bottom of the file and to focus on the test immediately.